articles/client-platforms/angular2/02-custom-login.md

```diff
@@ -90,7 +93,7 @@ Now, let's add a form to actually call login:
 
 ## Sign up
 
-To do a Sign Up just call `signup` method on `auth0` instance:
+To allow Sign-up, call the `signup` method on the `auth0` instance:
```

Comment:
To allow Sign-up, call the `signup` method => To allow users to sign up, provide a `signUp` method